crates/matrix-sdk/Cargo.toml

```diff
@@ -76,6 +77,7 @@ futures-core = "0.3.21"
 futures-signals = { version = "0.3.30", default-features = false }
```

Comment:
Can we remove it?

```suggestion
```

Edit: Once we have removed it from Sliding Sync too ;-).